benchmarks/render_lambda_benchmark.rb

```diff
@@ -0,0 +1,50 @@
+$:.unshift "lib"
+
+require "mustache"
+require "benchmark/ips"
+
+BASE_TEMPLATE = <<end_of_template
+<h2>Lambdas</h2>
+{{#lambdas}}
+  {{#lambda}}
+    Here is some text inside the lambda.\n
+    Also a variable is present: {{name}}.
+  {{/lambda}}
+{{/lambdas}}
+end_of_template
+
+one_name = [
+  {
+    name: "Charlie Chaplin",
+    lambda: lambda {|text| "\n--\n#{text}\n--\n" }
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/rokusu/mustache/file/2024324643/issues/source?bid=2903613&fileBranchId=3121922#l19)